### ISSUE TITLE
Bot/Crawler version is not parsing

### ISSUE THREAD

**Issue by amytymail** [matomo-org/device-detector#6291]
Hi team,

While parsing a bot why are we not parsing the bot version.
for example:
Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)

here we have Google bot which has version 2.1, can we add version number for bots too.

Thanks

**Comment by sgiehl** [comment]
Guess we could do so. But actually I'm wondering what that information could be good for.

**Comment by amytymail** [comment]
Well it is piece of info we are getting from the UA string and for me I want to show the User each n every piece of info we are getting from his UA string. 

Now it totally depends on. the User what he will do with that info.

Thanks

**Comment by liviuconcioiu** [comment]
> 
> 
> Guess we could do so. But actually I'm wondering what that information could be good for.

This information should be good only for SEOs. For example different variants of the Google bot, parse JavaScript differently.

**Comment by sgiehl** [comment]
Well we are grouping different types of Google bots into one at the moment. See https://github.com/matomo-org/device-detector/blob/master/regexes/bots.yml#L652
and I don't think we should split that up into multiple detections. So adding a version would be difficult. But if anyone is keen on doing that. Feel free to create a PR for that

**Comment by sanchezzzhak** [comment]
bots will always have the latest version

**Comment by sgiehl** [comment]
I don't think it brings much benefit to add that. If you need that @amytymail feel free to implement it or create our own Parser class doing that.
